apps/design-land/src/app/button/button.module.ts

```diff
@@ -16,9 +23,31 @@ import { FontAwesomeModule } from '@fortawesome/angular-fontawesome';
     CommonModule,
 
     DesignLandButtonRoutingModule,
+    DesignLandExampleViewerModule,
     DaffButtonSetModule,
     DaffButtonModule,
-    FontAwesomeModule
+    DaffArticleModule,
+    ButtonExamplesModule
   ]
 })
-export class ButtonModule {}
+export class ButtonModule {
+  constructor(
+    injector: Injector, 
+    private componentFactoryResolver: ComponentFactoryResolver
+  ) {
+    BUTTON_EXAMPLES
+      .map((classConstructor) => {
+        return {
+          element: createCustomElement(classConstructor, {injector}),
+          class: classConstructor
+        }
+      })
+      .map((customElement) => {
+        // Register the custom element with the browser.
+        customElements.define(
+          this.componentFactoryResolver.resolveComponentFactory(customElement.class).selector + '-example', 
```

Comment:
`resolveComponentFactory()` should be `resolveComponentFactory<unknown>()` to prevent a typescript bug. There is an example of this fix in design-land's `radio.module`.

Reply:
fixed